bootstrap/add-font-awesome-icons-to-all-of-our-buttons.html

```diff
@@ -0,0 +1,54 @@
+<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+
+    <style>
+        h2 {
+            font-family: Lobster, Monospace;
+        }
+
+        .thick-green-border {
+            border-color: green;
+            border-width: 10px;
+            border-style: solid;
+            border-radius: 50%;
+        }
+    </style>
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-8"><h2 class="text-primary text-center">CatPhotoApp</h2></div>
```

Comment:
Adjust div blocks to keep coding cleaner and more readable.
Ex:
<div>
    <h2>....</h2>
</div>